### DISCUSSION TITLE
launch an app on windows 10 after WOL?

### DISCUSSION THREAD

**Discussion by ropi59** [AlexGustafsson/homebridge-wol#151] [Questions]
all is in the title. Is it possible to launche a specific app after WOL on win 10?
for example i want to say "siri launch film" and  my win 10 pc WOL and launch KODI.
thanks

**Comment by AlexGustafsson** [comment]
Yes and no. I don't think you will be able to use the keyword `launch`, but you may be able to use `Siri turn on film`. You'll need to create a configuration like the following one:

```
{
  "accessory": "NetworkDevice",
  "name": "film",
  "startCommand": "ssh -flags -flags user@host /path/to/script"
}
```

Then you will have to create a script on your PC that starts Kodi. How to do that is out of scope for this project, though. Please refer to https://superuser.com or other forums for help on that.

**Reply by ropi59** [reply]
thanks for your response. i have found tuto to create a shortcut to launch app. would know you or put this shortcut to execute it with homebridge?
